module/mailer/src/Infrastructure/Sender/MailerSender.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Mailer\Infrastructure\Sender;
+
+use Ergonode\Mailer\Domain\MailInterface;
+use Webmozart\Assert\Assert;
```

Comment:
If needed maybe `webmozart/assert` should be a direct dependency. Looks like it's not